test/charts/constraints-chart/templates/NOTES.txt

```diff
@@ -0,0 +1,22 @@
+1. Get the application URL by running these commands:
```

Comment:
Remove NOTES.txt and _helpers from helm charts if not needed.
And leave values.yaml empty to not introduce unneeded confusion.